xpdacq/beamtime.py

```diff
@@ -77,6 +77,25 @@ def _configure_pe1c(exposure):
 
 
 def ct(dets, exposure, *, md=None):
+    """
+    Take one reading from area detectors with given exposure time
+
+    Parameters
+    ----------
+    detectors : list
+        list of 'readable' objects
+    exposure : float
+        total time of exposrue in seconds
+    md : dict, optional
+        extra metadata
+
+    Note
+    ----
+    area detector that is triggered will always be the one configured in
```

Comment:
how does this relate to `dets` that is passed as an argument though?  It makes it confusing what is happening